R/checkMissings.R

```diff
@@ -2,25 +2,37 @@
 #############################################################################
 #' Check and Adjust Missing Coding
 #'
-#' Function to check if missings are coded and labeled correctly in a \code{GADSdat} object.
+#' Functions to check if missings are coded and labeled correctly in a \code{GADSdat} object.

```

Comment:
Nit: You could also use \`GADSdat\` to show code.

Reply:
was not aware of that feature, nice!

Reply:
(will not be implemented due to incompatabilities):
[roxygen2](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd-formatting.html)